docs/using-hive.md

```diff
@@ -157,6 +157,102 @@ Credentials will be read from `~/.gcp/osServiceAccount.json`, this can be create
 bin/hiveutil create-cluster --base-domain=mydomain.example.com --cloud=gcp mycluster
 ```
 
+#### Create Cluster on Bare Metal
+
+Hive supports bare metal provisioning as provided by [openshift-install](https://github.com/openshift/installer/blob/master/docs/user/metal/install_ipi.md)
+
+At present this feature requires a separate pre-existing libvirt provisioning host to run the bootstrap node. This host will require very specific network configuration that far exceeds the scope of Hive documentation. See *LINK TBD* for more information.
+
+![Bare metal provisioning with one hypervisor host per cluster](hive-baremetal-hypervisor-per-cluster.png)
+
+![Bare metal provisioning with a shared hypervisor host](hive-baremetal-shared-hypervisor.png)
+
+To provision bare metal clusters with Hive:
+
+Create a Secret containing a [bare metal enabled InstallConfig](https://github.com/openshift/installer/blob/master/docs/user/metal/install_ipi.md#install-config).
```

Comment:
We could mention here that the install-config must include the libvirtURI, unless you think that's implicit based on line 164?

Reply:
Added.